frontend/src/k8sTypes.ts

```diff
@@ -365,3 +375,24 @@ export type AWSSecretKind = SecretKind & {
     };
   };
 };
+
+export type AccessReviewResourceAttributes = {
+  group?: string;
+  resource?: string;
+  subresource?: string;
+  verb?: K8sVerb;
+  name?: string;
+  namespace?: string;
+};
```

Comment:
Let us elaborate for our own uses / documentation.
```suggestion
export type AccessReviewResourceAttributes = {
   /** CRD group, '*' for all groups, omit for core resources */
  group?: '*' | string;
   /** Plural resource name, omit for all */
  resource?: string;
  /** TODO: Not a full list, could be expanded */
  subresource?: 'spec' | 'status';
  /** Must provide the verb you are trying to do; '*' means all verbs */
  verb: '*' | K8sVerb;
  /** A resource name, omit when not interested in a specific resource */
  name?: string;
  /** The namespace the check is in, omit for unbounded check */
  namespace?: string;
};
```